projects/step-core/src/lib/modules/custom-forms/components/custom-form/custom-form.component.scss

```
@@ -9,4 +9,8 @@
   &:not(.inline) {
     flex-direction: column;
   }
+
+  .flex-1 {
+    flex: 1;
+  }
```

- It make sense for inline forms only.
- Avoid class names, that duplicates style rules

```suggestion
  .inline.fill-whole-space {
    flex: 1;
  }
```